openmicroscopy/openmicroscopy
# Disable orphaned container

This PR replace https://github.com/openmicroscopy/openmicroscopy/pull/4431

Allowing to disable orphaned container as tweaked in https://github.com/openmicroscopy/openmicroscopy/pull/4246/

Test defaults:
 - if you tested that PR before unset ``bin/omero config set omero.client.ui.tree.orphans ``otherwise you shouldn't touch it.
 - test own and someone else data as user, group owner and admin. you should see "Orphaned container" everytime 

Test custom name via old properties:
 - set ``bin/omero config set omero.client.ui.tree.orphans.name 'Trash'`` and ``bin/omero config set omero.client.ui.tree.orphans.description "This is trash."'``
 - test if custom name works

Test disabled:
**you can only disable while viewing other users data as regular user. For admin and group owners container will be present**
 - set ``bin/omero config set omero.client.ui.tree.orphans.enabled False``
 - load your own tree, default "Orphaned Images" should be present,
 - chose someone else data, tree should not load orphaned container
 - cut image if you have permissions

cc: @dominikl Insight will need to take these to the account at some point
